.github/workflows/docs.yml

```diff
@@ -0,0 +1,28 @@
+name: Generate Docs
```

Comment:
@dherman @amilajack Do we use Github Pages for docs anymore? I thought we switched to only using `docs.rs`?

Reply:
Yea we do. I think the domain is hosted with netlify. But our static content for the site is hosted with gh pages. Cargo generated API docs are hosted on `docs.rs`.